### PR TITLE
Added testing for RBAC functionality of INSERT privilege.

### PR DESCRIPTION
**Pull request by MyroTk**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Added tests for RBAC functionality of INSERT privilege in TestFlows.

Changelog category:
- Build/Testing/Packaging Improvement

Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
Added tests for RBAC functionality of INSERT privilege in TestFlows.

Detailed description / Documentation draft:
Added tests for RBAC functionality of INSERT privilege in TestFlows.

### CODE REVIEW

File: tests/testflows/rbac/configs/clickhouse/common.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<yandex>
+    <timezone>Europe/Moscow</timezone>
+    <listen_host replace="replace">0.0.0.0</listen_host>
```
Comment (qoega) [inline tests/testflows/rbac/configs/clickhouse/common.xml:3]: You should be ready that tests can be run on ipv6only hosts.
```
                 2ms                [clickhouse1] 2020.07.29 01:53:03.101271 [ 1 ] {} <Warning> Application: Listen [::1]:8123 failed: Poco::Exception. Code: 1000, e.code() = 99, e.displayText() = Net Exception: Cannot assign requested address: [::1]:8123 (version 20.7.1.4205). If it is an IPv6 or IPv4 address and your host has disabled IPv6 or IPv4, then consider to specify not disabled IPv4 or IPv6 address to listen in <listen_host> element of configuration file. Example for disabled IPv6: <listen_host>0.0.0.0</listen_host> . Example for disabled IPv4: <listen_host>::</listen_host>

```

Comment (MyroTk) [reply tests/testflows/rbac/configs/clickhouse/common.xml]: Updated to use `::` instead of `0.0.0.0` to allow different hosts

**Comment by vzakaznikov** [comment]
@vitlibar, this PR is ready for your review.

**Comment by vitlibar** [comment]
There are conflicts, could you resolve them please?

**Pull request closed without merging**